Tools/scripts/freeze_modules.py

```diff
@@ -34,7 +34,7 @@
             sys.exit("ERROR: missing _freeze_module")
 else:
     def find_tool():
-        for arch in ['amd64', 'win32']:
+        for arch in ['amd64', 'win32', 'arm64']:
```

Comment:
This change is fine, but should probably also check whether it's running in an ARM64 build of Python first. If this is an x86/x64 build, there's no point trying to launch the ARM64 binary.